alby-nostr-wallet-connect/umbrel-app.yml

```diff
@@ -2,32 +2,32 @@ manifestVersion: 1
 id: alby-nostr-wallet-connect
 name: Nostr Wallet Connect
 tagline: The power of the zaps at the tip of your fingers.
-category: finance
-version: "0.3.0"
+icon: https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_icon.svg
```

Comment:
This entire line can be removed, since as an app on the umbrel app store the icon is hosted here: https://github.com/getumbrel/umbrel-apps-gallery/tree/master/alby-nostr-wallet-connect